unity-client/Assets/ABConverter/Core.cs

```diff
@@ -83,20 +85,20 @@ public Core(Environment env, Client.Settings settings = null)
         /// </summary>
         /// <param name="rawContents">A list detailing assets to be dumped</param>
         /// <param name="OnFinish">End callback with the proper ErrorCode</param>
-        public void Convert(ContentServerUtils.MappingPair[] rawContents, Action<ErrorCodes> OnFinish = null)
+        public void Convert(ContentServerUtils.MappingPair[] rawContents, Action<ErrorCodes> OnFinish = null, bool clearDirectoriesOnStart = true)
```

Comment:
suggestion: we can add `clearDirectoriesOnStart` to the Settings struct instead of passing it like this.

Reply:
Totally, I'll change that, good idea

Reply:
Done!